src/webhooks/overview.mdx

```diff
@@ -0,0 +1,74 @@
+---
+name: Overview
+menu: WEBHOOKS
+route: /docs/webhooks/overview
+---
+
+import { Title, Code, TextBlock } from "components"
+import webhook_settings from 'assets/webhook_settings.png'
+
+<Title>Overview</Title>
+Webhooks are sent from anomalies. Anomalies are described <a href="https://updates.smartlook.com/worried-about-anomalies-in-your-events-190172" target="_blank">HERE</a>
+
+# Cookbook
+
+<h3> 1. Set webhook URL on anomalies </h3>
+
+Optionaly set secret
```

Comment:
Nesel by tady pridat link a mozna jeste popis, kde to vlatne v dashboardu najdou?

Reply:
dal jsem link a screen :)